lwgenerate/hcl.go

```diff
@@ -204,6 +210,12 @@ func setBlockAttributeValue(block *hclwrite.Block, key string, val interface{})
 			data[attrKey] = value
 		}
 		block.Body().SetAttributeValue(key, cty.ObjectVal(data))
+	case map[string]string:
+		value, err := convertTypeToCty(v)
+		if err != nil {
+			return err
+		}
+		block.Body().SetAttributeValue(key, value)
```

Comment:
@ipcrm I've added explicit support for `map[string]string` rather than using `map[string]interface`. I want to be explicit that the labels can only be string and not allow values of any other type. Does this change seem reasonable at the hcl.go level?

Reply:
@rmoles I don't have any issue with this; I see how your using it and it makes sense to me to be able to set `map[string]string` and not have to do any additional validation.